build/ProjectReunion-CI.yml

```diff
@@ -50,8 +55,9 @@ jobs:
 - template: AzurePipelinesTemplates\ProjectReunion-CreateNugetPackage-Job.yml
   parameters:
     jobName: CreateNugetPackage
-    dependsOn: Build
+    dependsOn: PublishMRT
     prereleaseVersionTag: ci
+    condition: Always()
```

Comment:
>condition: Always() [](start = 4, length = 19)

shouldn't do only when succeeded?

Reply:
Geoffrey, was this the weird "wouldn't run" problem we were looking at before?  Does seem like we should remove always as a condition.

Reply:
Ah, yes.  I still haven't figured out why when this condition runs, succeeded() and succeededOrFailed() both return false.

Reply:
https://dev.azure.com/ms/ProjectReunion/_build/results?buildId=114742&view=results

Here's what happens when I change it to only run on succeeded().  The same thing happens if I use succeededOrFailed().  I'm not sure why, but Always() is the only condition that is working.

Reply:
why do you put condition inside parameters ? :)

Reply:
Since the dependsOn was parameterized, I figured the condition should be as well.

Reply:
oh, you have a parameter whose name collides with a keyword. Not sure whether it's a good practice. Try not to pass condition in, but check succeeded() inside CreateNugetPackage directly and see if it makes any difference. In the end, succeeded() check is default and unnecessary.

Reply:
Same result unfortunately.
https://dev.azure.com/ms/ProjectReunion/_build/results?buildId=114749&view=results

Reply:
no clue. I'd say just remove condition as succeeded should be the default behavior.

Reply:
I believe that will also result in the job being skipped.  I'm pretty confused about why that is, since the previous job that CreateNugetPacakge depends clearly succeeded.

I'm running it without any condition here:
https://dev.azure.com/ms/ProjectReunion/_build/results?buildId=114751&view=results

Reply:
Solution was to make the condition specifically look at the PublishMRT job.  I'm not sure why this is, but it works.

Reply:
Is this task going away soon?  I'm wondering if this going to come back ... (no action requested from this comment).

Reply:
Is this job going away soon?  I don't think so.  I think even once we start onboarding more sub-modules, this job will stay mostly as-is.